vmware_aria_operations_integration_sdk/containerized_adapter_rest_api.py

```diff
@@ -136,7 +136,7 @@ async def get_request_body(project: Project, connection: Connection) -> Dict:
             "credentialFields": fields,
         }
 
-    request_body = {
+    request_body: Dict[str, object] = {
```

Comment:
In a number of places, we're using `Dict[str, Any]` for the type of python "json". It turns out that has some limitations in mypy, and we should switch to `Dict[str, object]`, which will type-check better. I did that here because we're using it, but not elsewhere in the project.